docs/admin/config.rst

```diff
@@ -611,7 +611,7 @@ DEFAULT_MERGE_STYLE
 
 .. versionadded:: 3.4
 
-Merge style for any new components.
+Merge-style for any new components.
```

Comment:
It is called "Merge style" in all other places. If you intend to change this, please change it in all places at once in a separate pull request for each such change.

Reply:
How about referencing the actual setting instead?

```suggestion
:ref:`component-merge_style` for any new components.
```

Reply:
Seems better to me. What I wanted to avoid was that it possibly reads like "Merge (the) style for any new components", rather than "(the) merge style for any new components."